compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/viewinterop/InteropContainer.skiko.kt

```diff
@@ -104,6 +104,31 @@ internal fun InteropContainer.countInteropComponentsBelow(holder: InteropViewHol
     return componentsBefore
 }
 
+/**
+ * Sort the given interop components by their order in the container.
+ *
+ * @param holders The list to sort.
+ * @return The sorted list.
+ */
+internal fun InteropContainer.interopComponentsSortedByDrawOrder(
+    holders: Collection<InteropViewHolder>
+): List<InteropViewHolder> {
+    val remainingHolders = holders.toMutableSet()
+    val result = mutableListOf<InteropViewHolder>()
+    rootModifier?.traverseDescendantsInDrawOrder {
+        val holder = it.interopViewHolder
+        if ((holder != null) && remainingHolders.remove(holder) && contains(holder)) {
```

Comment:
`contains` is lookup in the same collection

```suggestion
        if (holder != null && remainingHolders.remove(holder)) {
```

Reply:
Done.